saltproc/version.py

```diff
@@ -3,8 +3,8 @@
 
 # Format expected by setup.py and doc/source/conf.py: string of form "X.Y.Z"
 _version_major = 0
-_version_minor = 3
-_version_micro = '1'  # use '' for first of series, number for 1 and above
+_version_minor = 4
+_version_micro = '0'  # use '' for first of series, number for 1 and above
```

Comment:
Why do you have a 0 here? Based on the comment, shouldn't it be `''`?

Reply:
I think I tried that once and it didn't work. I can check though

Reply:
I read over this again and the reason we have a 0 there is because we are also using the `_version_extra` to signify that this version is in development and not a release version.

Reply:
Would you mind changing the comment to reflect this?

Reply:
I read this over again and now I agree with you